src/nav/serverless-function-monitoring.yml

```diff
@@ -25,6 +25,8 @@ pages:
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-java
           - title: Trace your .NET Lambda functions with New Relic and OpenTelemetry
             path: /docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet
+          - title: OTEL log ingestion solution for ADOT lambda layer collector
```

Comment:
If at all possible, try to create a short version of the title that only goes on two lines